data/json/furniture_and_terrain/terrain-manufactured.json

```
@@ -447,6 +447,29 @@
       ]
     }
   },
+  {
+    "type": "terrain",
```

```suggestion
    "type": "terrain"
```
Making this change to rebase failed tests